source/utils/deprecation-warning.ts

```diff
@@ -0,0 +1,14 @@
+const alreadyWarned: string[] = [];
+
+export default (message: string) => {
+	if (alreadyWarned.includes(message)) {
+		return;
+	}
+
+	alreadyWarned.push(message);
+
+	// @ts-ignore
+	process.emitWarning(message, {
+		type: 'DeprecationWarning'
```

Comment:
```suggestion
		type: 'GotDeprecationWarning'
```

Reply:
?

Reply:
By emitting a warn with type: `DeprecationWarning` Node.JS sees it as a deprecation.
Therefore you can use `--trace-deprecation`.

Reply:
Or maybe `Got/DeprecationWarning`?

Reply:
It only works with exactly "DeprecationWarning".